rules/xcodeproj.bzl

```diff
@@ -2,7 +2,25 @@
 
 load("//rules:legacy_xcodeproj.bzl", legacy_xcodeproj = "xcodeproj")
 load("@xchammer//:BazelExtensions/xcodeproject.bzl", xchammer_xcodeproj = "xcode_project")
-load("@xchammer//:BazelExtensions/xchammerconfig.bzl", "project_config")
+load("@xchammer//:BazelExtensions/xchammerconfig.bzl", "bazel_build_service_config", "project_config")
+load("//rules:library.bzl", "GLOBAL_INDEX_STORE_PATH")
+
+def _patch_bazel_build_service_config(kwargs_bazel_build_service_config):
+    """Overriding to set some default values for now.
+
+    We need to better expose the option to
+    customize `GLOBAL_INDEX_STORE_PATH` and expecting to iterate a bit on `indexing_data_dir`
+    and run some tests, once it's battle tested will revert this and allow customers to fully configure
+    """
+    if kwargs_bazel_build_service_config:
+        return bazel_build_service_config(
+            bep_path = kwargs_bazel_build_service_config.bepPath,
+            index_store_path = "$SRCROOT/%s" % GLOBAL_INDEX_STORE_PATH,
+            indexing_data_dir = "$SRCROOT/xcbuildkit-data/indexing",
```

Comment:
Any way to we sneak this into an existing known directory or derived data path vs making a new top level dir? I was wondering what the signnificance of the symlink is in the other comment VS just handing it the `GLOBAL_INDEX_STORE_PATH`

Reply:
Yeah, moved the DataStore setup to `xcbuildkit` now, see latest commit there. Re `xcbuildkit-data` it's simple to move it to a different place, only added it here since we already had he `xchammer` one. But it's not a requirement for it to live under `$SRCROOT`.

I'll move it to a path under DerivedData, I think that works well.

Reply:
I might put this under `/tmp` for now